app/javascript/components/BubbleChart.js

```diff
@@ -111,7 +111,7 @@ const createRow = (dataRow, title, sectors) => {
       </div>
       {dataRow.map((el, i) => {
         const companiesBubbles = el.map(company => ({
-          value: COMPANIES_SIZES[company.size],
+          value: COMPANIES_SIZES[company.market_cap_group],
```

Comment:
I'd move the denormalisation outside jsx because it will happen at every component render, which can happen due to any higher order component state change.

Reply:
I don't know what that means :$ I just renamed things.

Reply:
Gotchu. Opened the file and it is a bit more complex than it looked.
Nevermind my comment, i'll look at it in another PR.